src/category_theory/limits/shapes/pullbacks.lean

```diff
@@ -360,21 +314,15 @@ end pushout_cocone
 
     If you're thinking about using this, have a look at `has_pullbacks_of_has_limit_cospan`,
     which you may find to be an easier way of achieving your goal. -/
+@[simps]
 def cone.of_pullback_cone
   {F : walking_cospan.{v} ⥤ C} (t : pullback_cone (F.map inl) (F.map inr)) : cone F :=
 { X := t.X,
-  π :=
-  { app := λ X, t.π.app X ≫ eq_to_hom (by tidy),
-    naturality' := λ j j' g,
-    begin
-      cases j; cases j'; cases g; dsimp; simp,
-      exact (t.w inl).symm,
-      exact (t.w inr).symm
-    end } }.
+  π := t.π ≫ (diagram_iso_cospan F).inv }
 
-@[simp] lemma cone.of_pullback_cone_π
+@[simp] lemma cone.of_pullback_cone_π_app
```

Comment:
This simp lemma is redundant since it's implied by the new simps above and another simp lemma.